src/utils/tasksUtils.ts

```diff
@@ -51,11 +51,18 @@ export async function getRunningQuarkusDevTasks(workspaceFolder: WorkspaceFolder
 export function getTaskExecutionWorkingDir(task: Task): string|undefined {
   if (!task.execution) return undefined;
 
-  if (!task.execution.options || !task.execution.options.cwd) {
+  const execution: ProcessExecution | ShellExecution | CustomExecution = task.execution;
```

Comment:
These changes are for removing a compilation error that's present in the master branch. Maybe they should be moved to a different PR.